src/everest/util/__init__.py

```diff
@@ -76,11 +77,38 @@ def makedirs_if_needed(path, roll_if_exists=False):
     os.makedirs(path)
 
 
+def warn_user_that_runpath_is_nonempty() -> None:
+    question = (
+        "You choose to run Everest in an existing non-empty runpath.\n"
+        "Please be aware of the following:\n"
+        "- Previously generated results "
+        "might be overwritten.\n"
+        "- Previously generated files might "
+        "be used if not configured correctly.\n"
+        "Are you sure you want to continue? Y/N : "
+    )
+    if not _get_user_confirmation(question):
+        sys.exit(
+            "Everest terminated execution due to existing runpath folder.\n"
+            "Please manually delete or change simulation_folder path in the config.yml"
+        )
+
+
+def _get_user_confirmation(message: str) -> bool:
+    try:
+        result: str = input(message)
+        return bool(result.upper() in ["Y", "YES"])
+
+    except Exception as e:
+        logging.getLogger(EVEREST).error(f"Exception {e}: Unable to parse user input")
+        raise e
+
+
 def _roll_dir(old_name):
     old_name = os.path.realpath(old_name)
     new_name = old_name + datetime.datetime.utcnow().strftime("__%Y-%m-%d_%H.%M.%S.%f")
     os.rename(old_name, new_name)
-    logging.getLogger("everest").info("renamed %s to %s" % (old_name, new_name))
+    logging.getLogger(EVEREST).info("renamed %s to %s" % (old_name, new_name))
```

Comment:
Would put this part in a separate commit unless it somehow relates to the issue you are on now. Also in that separate commit, feel very free to replace the old `%s` format with modern python fstring `f"...{}.."`